.github/workflows/dev-checks.yml

```diff
@@ -228,28 +228,45 @@ jobs:
               const excludedDirectories = ['.github','.gitignore'];
               const modifiedFiles = response.data.files;
               const modifiedDirectories = {};
+              const newDirectories = [];
+
               response.data.files.forEach(file => {
                 const fileDirectory = file.filename.split("/")[0];
-                 if (!excludedDirectories.includes(fileDirectory)) {
-                if (!modifiedDirectories[fileDirectory]) {
-                  modifiedDirectories[fileDirectory] = false;
-                }}
+                if (!excludedDirectories.includes(fileDirectory)) {
+                  if (!modifiedDirectories[fileDirectory]) {
+                    modifiedDirectories[fileDirectory] = false;
+                    if (!excludedDirectories.includes(fileDirectory) && !newDirectories.includes(fileDirectory)) {
```

Comment:
Remove `! excludedDirectories.includes(fileDirectory) &&` part. This is redundant with line 235

Reply:
sure